apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -589,6 +619,14 @@ def generate_project(self, model_library_format_path, standalone_crt_dir, projec
 
                     cmake_f.write(line)
 
+                heap_size = _get_recommended_heap_size(options)
+                if options.get("heap_size"):
+                    board_mem_size = _get_board_mem_size(options)
+                    if board_mem_size is not None:
+                        assert options["heap_size"] < board_mem_size, "heap size is too large"
```

Comment:
make error message more clear? `Heap size (options["heap_size"]) is larger than memory size (board_mem_size) on this board.`

Reply:
done